int32[6] durations
---
bool status